lib/preprocessor.h

```diff
@@ -154,12 +154,6 @@ class CPPCHECKLIB Preprocessor {
      */
     std::string getcode(const std::string &filedata, const std::string &cfg, const std::string &filename);
 
-    /**
-     * preprocess all whitespaces
-     * @param processedFile The data to be processed
-     */
-    static void preprocessWhitespaces(std::string &processedFile);
```

Comment:
The usage was removed in 34322573908f9cb3dd09a8d77bd82d0c8d8c7fe9 - leftover code from removed feature so it can be deleted.